src/Tasks.UnitTests/Unzip_Tests.cs

```diff
@@ -134,7 +134,7 @@ public void LogsErrorIfReadOnlyFileCannotBeOverwitten()
 
                 unzip.Execute().ShouldBeFalse(() => _mockEngine.Log);
 
-                _mockEngine.Log.ShouldContain("D6DFD219DACE48F8B86EFCDF98433333.txt' is denied", () => _mockEngine.Log);
+                _mockEngine.Log.ShouldContain($"D6DFD219DACE48F8B86EFCDF98433333.txt{ (NativeMethodsShared.IsMono ? "\"" : "'") } is denied", () => _mockEngine.Log);
```

Comment:
Is the difference in string coming from MSBuild or some other library with a slightly-different implementation on Mono?

Reply:
It's from mono. And as the implementations converge, this should go away.